launcher/ui/themes/ThemeManager.cpp

```diff
@@ -55,55 +57,51 @@ void ThemeManager::InitializeThemes() {
         auto searchPaths = QIcon::themeSearchPaths();
         searchPaths.append("iconthemes");
         QIcon::setThemeSearchPaths(searchPaths);
-        themeDebugLog << "<> Icon themes initialized.";
+        themeDebugLog() << "<> Icon themes initialized.";
     }
 
     // Initialize widget themes
     {
-        themeDebugLog << "<> Initializing Widget Themes";
-        themeDebugLog "✓ Loading Built-in Theme:" << AddTheme(new SystemTheme());
-        auto darkThemeId = AddTheme(new DarkTheme());
-        themeDebugLog "✓ Loading Built-in Theme:" << darkThemeId;
-        themeDebugLog "✓ Loading Built-in Theme:" << AddTheme(new BrightTheme());
+        themeDebugLog() << "<> Initializing Widget Themes";
+        themeDebugLog() << "Loading Built-in Theme:" << AddTheme(std::make_unique<SystemTheme>(SystemTheme()));

```

Comment:
same here, just `std::make_unique<SystemTheme>()`
and same for the following ones